### PR TITLE
Fix for PLD AF3 quest mobs not spwaning

### PR DESCRIPTION
**Pull request by m3rlin87**
Added a script file for each of the two mobs that spawn from the village well in Davoi for PLD AF3 quest.
Updated mob_spawn_point sql file to contain spawn points of the two mobs to be around the village well

======

This my first pull request and new to using github, let me know if I should do things differently in the future. I am guessing someone will say something about the number of commits as you all have to others before :) , but couldn't see a way to reduce them. Currently just using the web interface, tell i figure out a better way for my setup.

### CODE REVIEW

**Comment by xdemolish** [comment]
best to checkout a local copy and work on it from there
https://guides.github.com/activities/contributing-to-open-source/

once you've done that, you'll need to rebase/squash
https://help.github.com/articles/about-git-rebase/

File: scripts/zones/Davoi/mobs/One-eyed_Gwajboj.lua [hunk 1]
```diff
@@ -0,0 +1,21 @@
+-----------------------------------
+-- Area: Davoi
+-- NPC: One-Eyed Gwajboj
+-- Involved in Quest: Under Oath : PLD AF3
+-----------------------------------
+
+require("scripts/globals/settings");
+
```
Comment (TeoTwawki) [inline scripts/zones/Davoi/mobs/One-eyed_Gwajboj.lua]: When you have a mob script that does literately nothing, it doesn't need to exist.

File: scripts/zones/Davoi/mobs/Three-eyed_Prozpuz.lua [hunk 2]
```diff
@@ -0,0 +1,21 @@
+-----------------------------------
+-- Area: Davoi
+-- NPC: Three-Eyed Prozpuz
```
Comment (TeoTwawki) [inline scripts/zones/Davoi/mobs/Three-eyed_Prozpuz.lua]: same here, script does nothing, don't need it. The mob will still pop without this empty script.

**Comment by m3rlin87** [comment]
Yeah wasn't quite sure on if the files were needed or not. As initially after i updated my DB the mobs still didn't spawn, so i added the mob files. The had to reboot for some other reason and they started working, but at that point still didn't know if it was just the updated value and or the files that made things worked. 

Good to know for the future, thanks!

**Comment by bendangelo** [comment]
Could you squash this then I can merge it in?

**Comment by m3rlin87** [comment]
Yeah trying to do that. Still leaning GIT, hopefully ill have it figured out in a little bit from links the others posted.

**Comment by m3rlin87** [comment]
I am at a loss for squashing the commits, keeping getting this "Cannot 'squash' without a previous commit" But if i commit i get that everything is update.....I have copy of my fork locally and trying to squash them there. However, i did the initial changes from this website solely, not sure if thats part of the issue.

**Comment by TeoTwawki** [comment]
Reset history to an earlier commit using "mixed" mode reset which will leave your files unchanged, and then create a new commit. Then force push it.

**Comment by TeoTwawki** [comment]
for tortoise: 
https://tortoisegit.org/docs/tortoisegit/tgit-dug-reset.html and tick the "unknown" box in the push dialog.

for command line:
`git reset [<mode>] [<commit>]`
`git push --force`

**Comment by TeoTwawki** [comment]
Sometimes doing it that way is easier for people than rebase/squash

**Comment by bendangelo** [comment]
What Teo said it probably easiest for most people.

* First find the last commit: `b06c6a6389742c7470ca38d37c8505634302d805`
* Git reset to the commit before that. `git reset --mixed b06c6a6389742c7470ca38d37c8505634302d805~1`
* Make a new commit `git commit -m "New commit"`
* Now you must force push because you change the branches history `git push -f`

**Comment by m3rlin87** [comment]
Ok i think i have the commits corrected now.

**Comment by TeoTwawki** [comment]
Yep looks good now.

**Pull request merged**